qi-doc/scribblings/interface.scrbl

```diff
@@ -138,23 +143,34 @@ The core entry-point to Qi from the host language is the form @racket[☯]. In a
                            (ε flow-expr flow-expr)
                            (effect flow-expr flow-expr)
                            apply
-                           literal
-                           (quote value)
-                           (quasiquote value)
-                           (quote-syntax value)
-                           (syntax value)
                            (qi:* expr ...)
                            (expr expr ... __ expr ...)
                            (expr expr ... _ expr ...)
                            (expr expr ...)
-                           expr]
+                           literal
+                           identifier]
+                [literal (code:line)
```

Comment:
Can a literal be "empty"? If not, I recommend omitting `(code:line)` here and letting the first value show up after the `=` in the doc (much like the current `switch-expr` docs).